Packages/com.unity.inputsystem/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -94,17 +104,163 @@ public void TDO_Actions_CanDriveUI()
         uiModule.leftClick = new InputActionProperty(leftClickAction);
         uiModule.middleClick = new InputActionProperty(middleClickAction);
         uiModule.rightClick = new InputActionProperty(rightClickAction);
+        uiModule.scrollWheel = new InputActionProperty(scrollAction);
 
         // Enable the whole thing.
         map.Enable();
 
+        yield return null;
```

Comment:
Why do we need to wait a frame? Think this warrants a comment.

Reply:
Added a comment.
The canvas doesn't iterate over the graphics objects to determine depth ordering until the next frame.  So returning null let's the graphics get ordered so the raycaster can hit them.